zerver/lib/test_runner.py

```diff
@@ -179,9 +179,9 @@ def addFailure(self, *args: Any, **kwargs: Any) -> None:
 
     def addSkip(self, test: TestCase, reason: str) -> None:
         TestResult.addSkip(self, test, reason)
-        self.stream.writeln("** Skipping {}: {}".format(full_test_name(test),
+        self.stream.writeln("** Skipping {}: {}".format(full_test_name(test),  # type: ignore
                                                         reason))
-        self.stream.flush()
+        self.stream.flush()  # type: ignore
```

Comment:
This stream thing is a bug with the typeshed project; I opened the bug upstream and made this link to it.